src/Microsoft.PowerShell.Commands.Management/resources/ComputerResources.resx

```diff
@@ -387,4 +387,10 @@
   <data name="InvalidParameterForCoreClr" xml:space="preserve">
     <value>The {0} parameter is not supported for CoreCLR.</value>
   </data>
+  <data name="CommandFailed" xml:space="preserve">
+    <value>Command failed with exit code {0}: {1}</value>
```

Comment:
An end user doesn't know `Restart/Stop-Computer` uses the `shutdown` native command, so the exit code doesn't seem to make much sense. Maybe we just report the error message?

Reply:
Agree, the `shutdown` command should be an implementation detail, will have it just emit the error message.